### ISSUE TITLE
Go through the codebase and improve accessibility using W3-ARIA 

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4108]
Complex web applications become inaccessible when assistive technologies cannot determine the semantics behind portions of a document or when the user is unable to effectively navigate to all parts of it in a usable way. We can increase the accessibility of the codebase a lot by adding some attributes in the HTML code. 

As the codebase can be huge, we can divide the issue into multiple small issues. To work on this you can take up any of the issues from the list below or can even create one of you find any areas where accessibility can be improved.

Some areas where accessibility can be improved are:
- [ ] Adding roles: Attaching a role gives assistive technologies information about how to handle each element. Following roles can be attached to the elements:
     - [ ] Alert (#4109)
     - [x] Button (#4110, #4111, #4112, #4113, #4114) 
     - [x] Tabindex (#4129)
     - [ ] Checkbox
     - [ ] Form
     - [ ] Heading
     - [ ] Img (#4130)
     - [ ] Input
     - [ ] List
     - [ ] Menu

 A list of all the roles can be found here: https://www.w3.org/TR/wai-aria/#role_definitions You can reply to this issue if you find any other roles that can be applied to the codebase and I can add it in the list. 

Meanwhile I would be preparing multiple issues for these.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-860

**Comment by stacytonui** [comment]
Hello, I'm looking for issues to work on, maybe I can work on the alerts?

**Comment by renujain31** [comment]
Hey Ya sure! I just created an issue for the same #4109 . You can comment there if you want to get it assigned.

**Comment by onyeepeace** [comment]
@renujain31 I will like to work an an issue as well.

**Comment by Ifycode** [comment]
Me too 🙌

**Comment by amycruz97** [comment]
can i work on this too

**Comment by codifyit** [comment]
@Angamanga Can i work on this ?

**Comment by Angamanga** [comment]
This is a collection-issue, please read in the description, there are links to more specified issues there!
